styles/colors.less

```diff
@@ -39,9 +39,12 @@
 @base08: #f2777a;
 @base09: #f99157;
 @base0A: #ffcc66;
-@base0F: #d27b53;
-@base0E: #ccaac8;
-@base0D: #6699cc;
-@base0C: #66cccc;
 @base0B: rgb(18, 255, 169);
-
+@base0C: #66cccc;
```

Comment:
Not goint to add those new Base color, i will only use 16 base colors , not anymore.